docs/high_level_design.adoc

```diff
@@ -229,12 +97,10 @@ For example, say there is a new update to support SR-IOV.
 == Testing strategy
 Gtest framework is used for unit testing and some functional testing. Completed end to end and functional testing will be developed by plugging in controller, agent and Mizar together under Jenkins CI/CD framework. We are planning to leverage gcov+lcov for code coverage analysis.
 
-== Reference
+== References

```

Comment:
For references, it is recommended to use the format as in this doc: 
https://github.com/futurewei-cloud/alcor/blob/master/docs/visionary_design/data_store.adoc

Reply:
Fixed.